### PR TITLE
Add `AppInfoHarvester` for collecting app-specific signals

### PR DESCRIPTION
**Pull request by mgutski**
The initial `AppInfoHarvester` implementation includes a single `userInterfaceStyle` property for retrieving the interface style for the app.

The Light/Dark mode toggling was introduced in iOS 13, hence on iOS 12 the `userInterfaceStyle` property always returns `light` value.

### CODE REVIEW

File: Sources/FingerprintJS/Harvesters/DataExchange/DeviceInfo.swift [hunk 1]
```diff
@@ -1,16 +1,10 @@
...
-//  Created by Petr Palata on 15.04.2022.
-//
-
-import CoreGraphics
 import Foundation
 
 public struct DeviceInfo {
     public let vendorIdentifier: UUID?
 
+    public let userInterfaceStyle: UserInterfaceStyle
```
Comment (petrpalata) [inline Sources/FingerprintJS/Harvesters/DataExchange/DeviceInfo.swift:6]: This property should have some kind of public documentation. I know the rest of the struct doesn't but that's an oversight.

File: Sources/FingerprintJS/Harvesters/DataExchange/UserInterfaceStyle.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+public enum UserInterfaceStyle: String {
```
Comment (petrpalata) [inline Sources/FingerprintJS/Harvesters/DataExchange/UserInterfaceStyle.swift:3]: Same here re docs.

**Review by petrpalata — CHANGES_REQUESTED** [review]
Missing some docs, otherwise looks good.

File: Sources/FingerprintJS/Harvesters/DataExchange/DeviceInfo.swift (hunk 1 above)
Comment (mgutski) [reply Sources/FingerprintJS/Harvesters/DataExchange/DeviceInfo.swift:7]: Added.

File: Sources/FingerprintJS/Harvesters/DataExchange/UserInterfaceStyle.swift (hunk 2 above)
Comment (mgutski) [reply Sources/FingerprintJS/Harvesters/DataExchange/UserInterfaceStyle.swift:4]: Added.

**Review by petrpalata — APPROVED** [review]

**Pull request merged**